chase/backend/prisma/custom_seeds/seed_munbw2024.ts

```diff
@@ -0,0 +1,323 @@
+// import { faker } from "@faker-js/faker";
+import { $Enums, PrismaClient } from "../generated/client";
+import { alpha2ToAlpha3 } from "../../../../util/countryCodeUtils";
+import delegationData from "./munbw2024_delegations.json";
+const prisma = new PrismaClient();
```

Comment:
We could use the exported db const from the db module here since its based on the app config but thats not really necessary, just nice to have